src/Location/Location.tsx

```diff
@@ -12,107 +12,115 @@ import wind from '../assets/wind.png'
-// 			<div className='info-box'>
-// 				<h4 className='info-box-header'>HAPPENING NOW</h4>
-// 				<p className='current-date'>{Date.now().toLocaleString()}</p>
-// 				<article className='current-weather'>
-// 					<div className='current-weather-left'>
-// 						<h5 className='current-temp'>{currentWeather.temp}&deg;</h5>
-// 						<img src={cloudyNight} alt='clouds covering moon for current sky' className='large-weather-icon'/>
-// 					</div>
-// 					<div className='current-weather-right'>
-// 						<span className='current-weather-right-top'>
-// 							<img src={wind} alt='wind icon for wind speed direction' className='small-weather-icon'/> 
-// 							<p className='current-wind'>{currentWeather.wind_speed} mph / {currentWeather.wind_direction}</p>
-// 						</span>
-// 						<span className='current-weather-right-bottom'>
-// 							<img src={waterdrop} alt='rain droplet icon for precipitation' className='small-weather-icon'/> 
-// 							<p className='current-precipitation'>{(currentWeather.precipitation) ? currentWeather.precipitation : 0}</p>
-// 						</span>
-// 					</div>
+const Location = ({locationData, currentWeather, weeklyWeather, currentAir}: LocationProps) => {
 
-// 				</article>
-// 			</div>
-// 			<div className='info-box air-quality'>
-// 				<h4 className='info-box-header'>AIR QUALITY</h4>
-// 				<table>
-// 					<tbody>
-// 						<tr>
-// 							<td>
-// 								<img 
-// 								src={lungs} 
-// 								alt='lungs icon for air quality' 
-// 								className="small-weather-icon"/> 
-// 								<p className='type'>Air Quality Index</p>
-// 								<p className='unit'>{currentAir.AQI}</p>
-// 							</td>
-// 							<td>
-// 								<img 
-// 									src={sun} 
-// 									alt='sun icon for UV index' 
-// 									className="small-weather-icon"/> 
-// 									<p className='type'>UV Index</p>
-// 									<p className='unit'>{currentWeather.uvi} of 10</p>
-// 							</td>
-// 						</tr>
-// 						<tr>
-// 							<td>
-// 								<img 
-// 									src={bee} 
-// 									alt='bee icon for allergies and pollen' 
-// 									className="small-weather-icon"/> 
-// 									<p className='type'>Allergens</p>
-// 									<p className='unit'>no allergy data yet!</p>
-// 							</td>
-// 							<td> 
-// 								<img 
-// 									src={eye} 
-// 									alt='eye icon for visibility' 
-// 									className="small-weather-icon"/> 
-// 									<p className='type'>Visibility</p>
-// 									<p className='unit'>{currentWeather.visibility / 5280}mi</p>
-// 							</td>
-// 						</tr>
-// 					</tbody>
-// 				</table>
-// 			</div>
-// 			<div className='info-box hourly-forecast'>
-// 				<h4 className='info-box-header'>HOURLY FORECAST</h4>
-// 			</div>
-// 			<div className='info-box weekly-forecast'>
-// 				<h4 className='info-box-header'>WEEKLY FORECAST</h4>
-// 			</div>
 
-// 		</section>
-// 	)
-// }
+	// const renderWeeklyForecast = 
```

Comment:
Probably will move to a component that way we have more control if data doesn't load correctly.

Reply:
Great idea. It will definitely make error handling/displaying much easier.